website/src/components/rsv/RsvSingleVariantPage.astro

```diff
@@ -62,7 +63,15 @@ const lineageFilterConfigs = getLineageFilterConfigs(
         organismsConfig={getDashboardsConfig().dashboards.organisms}
         client:only='react'
     />
-    <div class='flex flex-col gap-y-6' slot='dataDisplay'>
+    <div class='mx-[8px] flex flex-col gap-y-6' slot='dataDisplay'>
+        <div class='h-56 sm:h-32'>
+            <ComponentHeadline>Stats</ComponentHeadline>
+            <gs-statistics
+                numeratorFilter={JSON.stringify(variantFilter)}
+                denominatorFilter={JSON.stringify(baselineFilter)}
+                width='100%'
+                height='100%'></gs-statistics>
+        </div>
```

Comment:
I was thinking that maybe we could wrap those things in their own Astro (or React?) component (some styling div, the headline and the web component). In the end, we reuse the same blocks of code on every single variant page.

But I didn't think this through in full extent. We could do it separately somewhen later.

Reply:
In general a good idea. I think we should do this after https://github.com/GenSpectrum/dashboards/pull/357 since I also refactored the views there a little.